mautic/mautic
# Extended categories permissions

With this PR I add the support to customize categories with extended permissions (view own, view other, etc.). You can set specific permissions on all modules (emails, landing pages, etc.).

Steps to test:
- From administration panel, create a role with restricted categories permission (i.e. "view own" but not "view other") on a module (i.e. the landing pages module). Set the same permissions on the module, so a user with this role can access only on his own works.
- Assign this role to an user.
- Login with this user and create a landing page category.
- Login with your admin account and create a landing page category, too.
- Admin can see both categories but user can see only his own category.
